lib/node_modules/@stdlib/math/base/special/besselj0/README.md

```diff
@@ -83,16 +83,17 @@ v = j0( NaN );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var randu = require( '@stdlib/random/base/randu' );
-var j0 = require( '@stdlib/math/base/special/besselj0' );
+var uniform = require( '@stdlib/random/array/uniform' );
+var logEachMap = require( '@stdlib/console/log-each-map' );
+var besselj0 = require( '@stdlib/math/base/special/besselj0' );
 
 var x;
-var i;
 
-for ( i = 0; i < 100; i++ ) {
-    x = randu() * 10.0;
-    console.log( 'j0(%d) = %d', x, j0( x ) );
-}
+x = uniform( 100, 0.0, 100.0, {
```

Comment:
```suggestion
var x = uniform( 100, 0.0, 100.0, {
```

Reply:
This and the above changes should be replicated across the other packages you have updated below.